client/src/pages/Welcomepage.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import DefaultButton from '../components/DefaultButton';
+
+
+export default function WelcomePage({ open }) {
+
+    return (
+        <Wrapper open={open}>
+
+
+            <h2>YOBro! Lets get started by setting up your Brofile!</h2>
+
+            <a href='/createbrofile'><DefaultButton buttonText='Create YOBrofile'></DefaultButton></a>
```

Comment:
```suggestion
            <Link to='/createbrofile'>Create YOBrofile'</Link>
```

It is not recommended (and I'm amazed that it actually works) to use anchor tags. 
Please use the `Link` element from 'react-router-dom`instead of the anchor tag. 

You can still use the styling of your Button on the Link element to create the look of a button :)

Reply:
Using anchor tags will work while working on a local server but once you deploy it (e.g. to Vercel) this would no longer work. So please use the react router dom

Reply:
With `styled-components` you can use the styles of your button and just tell that you button component should be a different element, in this case `Link`. Take a look at this section of an article: https://www.joshwcomeau.com/css/styled-components/#misc-tips-and-tricks
```suggestion
           <DefaultButton as={Link} buttonText='Create YOBrofile' />
```